lib.php

```diff
@@ -291,3 +318,14 @@ function scormremote_pluginfile($course, $cm, $context, $filearea, $args, $force
 
     send_stored_file($file, $lifetime, 0, false, $options);
 }
+
+/**
+ * Manipulate the string to token replace the debug log level.
+ * @param string $layer3content
+ * @return void
+ */
+function applyloglevel(string &$filecontent) {
```

Comment:
also this function is in global scope so best to prefix it with mod_scormremote so it doesn't clash, or move it to a namespaced helper

Reply:
done - yeah. That was a bit silly.